2024-10-29 20:36:40,300 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-10-29 20:36:40,301 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\instabot\bot\bot.py
2024-10-29 20:36:40,302 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-10-29 20:36:40,856 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2024-10-29 20:36:41,279 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2024-10-29 20:36:41,788 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2024-10-29 20:36:42,226 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2024-10-29 20:36:44,097 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2024-10-29 20:36:44,097 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXEUB6eV5h7ViBTWcIKeRmHE53qkOtzaTUp3vXOQawXCcU7Zw-G0Dkz_13sVBfzDLOjJqr8/AfwwYt9oaM5kk9KObsgJZo9if2v2BECicVhqT8CO-HjNKiMi-Ymh5MEZnjxjwmvjR3S1t6s3FDL9NQ/","api_path":"/challenge/AXEUB6eV5h7ViBTWcIKeRmHE53qkOtzaTUp3vXOQawXCcU7Zw-G0Dkz_13sVBfzDLOjJqr8/AfwwYt9oaM5kk9KObsgJZo9if2v2BECicVhqT8CO-HjNKiMi-Ymh5MEZnjxjwmvjR3S1t6s3FDL9NQ/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2024-10-29 20:36:44,097 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2024-10-29 20:36:44,097 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2024-10-29 20:36:44,099 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
